api/routes.py

```diff
@@ -512,3 +512,11 @@ def loadstorm_verify(code):
 @app.route('/healthcheck.html')
 def health_check():
     return Response("", 200)
+
+@app.route("/images/<filename>")
+def static_image(filename):
+    directory = os.path.join(os.path.abspath(os.path.dirname(__file__)), "..", "resources", "images")
+    cache_timeout = ConfigurationSetting.sitewide(
+        app._db, Configuration.STATIC_FILE_CACHE_TIME
+    ).int_value
+    return flask.send_from_directory(directory, filename, cache_timeout=cache_timeout)
```

Comment:
I looked up the docs for `send_from_directory` and `send_file` and didn't see an explicit indication that it handles hack attempts like `/images/../../../etc/passwd`. But it does say `send_from_directory` is "secure" so I assume that's what it's referring to.

Reply:
Based on the `send_from_directory` docs I think `/images/<path:filename>` would be a better route.

Reply:
Also there's no test for this controller in this branch. Is it feasible to add one?

Reply:
This is a route and we don't have tests for any routes. Should I make a controller for it? The admin static files routes could use it as well.

Reply:
You're right, I mentally put this code in a controller. I do think it's complex enough to deserve a controller.